### PR TITLE
truncates htmlpart to something ES can handle and adds authparameters to modify_keys

### PR DESCRIPTION
**Pull request by Phrozyn**

### CODE REVIEW

File: mq/plugins/cloudtrail.py [hunk 1]
```diff
@@ -117,6 +118,10 @@ def onMessage(self, message, metadata):
         if not message['source'] == 'cloudtrail':
             return (message, metadata)
 
+        if 'requestparameters' in message['details']:
```
Comment (arcrose) [inline mq/plugins/cloudtrail.py:121]: A simpler way of writing these two lines would be to do something like this:
```py
if message.get('details', {}).get('requestparameters', {}).get('htmlpart') is not None:
```

File: mq/plugins/cloudtrail.py [hunk 2]
```diff
@@ -117,6 +118,10 @@ def onMessage(self, message, metadata):
...
+            if 'htmlpart' in message['details']['requestparameters']:
+                message['details']['requestparameters']['htmlpart'] = message['details']['requestparameters']['htmlpart'][0:4095]
```
Comment (arcrose) [inline mq/plugins/cloudtrail.py:123]: Could we define a constant that describes the meaning of the value 4095 here?

File: mq/plugins/cloudtrail.py (hunk 1 above)
Comment (Phrozyn) [reply mq/plugins/cloudtrail.py]: This isn't consistent with how the rest of our code is currently written, but I'll change it.

File: mq/plugins/cloudtrail.py (hunk 2 above)
Comment (Phrozyn) [reply mq/plugins/cloudtrail.py:129]: I'll add a comment to the code to help.

File: mq/plugins/cloudtrail.py (hunk 1 above)
Comment (arcrose) [reply mq/plugins/cloudtrail.py]: I wouldn't consider this required; just a suggestion.

**Pull request merged**